api/handlers/userhandler.py

```diff
@@ -100,6 +105,19 @@ def post(self):
     def _init_storage(self):
         self.storage = containerstorage.ContainerStorage('users', use_object_id=False)
 
+    def _cleanup_user_permissions(self, uid):
+        try:
+            config.db.collections.delete_many({'curator': uid})
```

Comment:
If a user curates a collection, and adds at least 1 other admin, I would expect that collection to still exist. But given that we're not exposing this in the UI (and only using this internally) maybe that's ok.